src/components/routes/Trips/TripsReceipt/TripsReceipt.tsx

```diff
@@ -146,10 +146,11 @@ const TripsReceiptsMeta = (props: Booking): JSX.Element => {
                   <span>
                     {addressLine1 && `${addressLine1}, `}
```

Comment:
don't we have a address formatter for this?

Reply:
fixed! updated the formatter with the newer one from backend